zml/tensor.zig

```diff
@@ -859,6 +859,49 @@ pub const Tensor = struct {
         return _result(new_shape, op.result(0));
     }
 
+    pub fn conv3d(
+        input: Tensor,
+        kernel: Tensor,
+        opts: struct {
+            window_strides: []const i64 = &.{ 1, 1, 1 }, //[time, height, width]
```

Comment:
I mean this is literally just calling `convolution`.
Either you call it directly or you make conv3d an actual helper by
simplifying the argument types to better match a conv3d: strides become [3]i64, padding: [3][2]i64,